docs/docs/installation_guide/installation/pre-installation/china-prerequisites.md

```diff
@@ -0,0 +1,13 @@
+---
+id: china-prerequisites
+title: China Regions Prerequisites
+sidebar_label: China Regions Prerequisites
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This section is just for China regions(Beijing, Ningxia and Hongkong), The initial prerequisites include preparing an ICP licensed domain.
+## Preparing an ICP licensed domain
+
+Prepare an [ICP][icp] licensed domain. The Service Workbench console is served via CloudFront distribution which is considered as an Internet information service. It is referred to as **Service Workbench Console Domain**.
+For Hongkong region, this part is not mandatory.
```

Comment:
If no ICP required for HK deployment, why adding it in this part?

Reply:
Remove it